feature-group-definitions/fetch-priority.yml

```diff
@@ -0,0 +1,12 @@
+spec:
```

Comment:
OK, looks like we don't allow features with multiple spec links yet. @ddbeck is it time to make that change then? Do you think we need to designate one the URLs the main one?

Reply:
I checked the specs again and think we should actually use these two URLs:
https://fetch.spec.whatwg.org/#request-priority
https://html.spec.whatwg.org/multipage/urls-and-fetching.html#fetch-priority-attributes

The first is the concept of fetch priority, and the second is about the HTML attributes that set priority. The BCD entries then point at the API surface directly.

Reply:
I'll add the entries in https://github.com/mdn/browser-compat-data/pull/19634 now.